wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -0,0 +1,343 @@
+
+- `Fade Out`
+- `No Video`
+- `10K`
+- `Taiko`
+
+---
+
+When writing out game modifiers for tournament articles, they must instead use upper camel case (omit the space and keep the letter casing as seen above).
+
+### Gameplay elements
+
+Gameplay elements must not be capitalised, unless it is a link that links to the article. The following is an example:
+
+```markdown
+In osu!standard, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
+```
+
+### Language names
+
+Language names must be capitalised. The following is an example:
+
+```markdown
+The `#spanish` chat channel is for those who speak Spanish.
+```
+
+### Proper nouns
+
+Proper nouns must be capitalised. The following is an example:
+
+```markdown
+Dean Herbert (also known as peppy) created osu! back in 2007.
+```
+
+### Trademarks
+
+*For rules regarding osu!, see: [osu!](#osu!)*
+
+The following trademarks must be spelt as follows (letter casing must match):
+
+- `Discord`
+- `Facebook`
+- `GitHub`
+- `Google`
+- `Reddit`
+- `Skype`
+- `Twitch`
+- `Twitter`
+- `YouTube`
+
+Trademarks must not be followed by the trademark or registered trademark symbols.
+
+## Date and time
+
+### Date formatting
+
+*For date formatting, see [Wikipedia's Manual of Style on Date formatting](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Formats)*
+
+Date formatting should be consistent throughout the entire article.
+
+### Time formatting
+
+Time formatting must be done using the following format:
+
+```markdown
+HH:MM TIMEZONE
+```
+
+Time is written in 24-hour format and any single digits must be prefixed with a zero. The timezone must be written immediately after the time and should be in UTC+0 for global events or in the timezone of the event for smaller events. Use `UTC` instead of `GMT`.
+
+Bad examples:
+
+```markdown
+3:30 PM UTC
+22:30 (UTC+7)
+11:30
+```
+
+Good examples:
+
+```markdown
+15:30 UTC
+22:30 UTC+7
+11:30 UTC-4
+```
+
+`UTC` (without an offset) is implied to be `UTC+0`. Either one may be used, but usage must be consistent. UTC with a 0 offset must use a plus symbol (`+`).
+
+### Date and time formatting
+
+When both date and time are used together, the date must be written first, followed by the time. The time must be in between parenthesis (`(` and `)`). The following are examples:
+
+Bad examples:
+
+```markdown
+October 25, 2016 at 11:45 UTC
+October 25, 2016 11:45 UTC
+```
+
+Good example:
+
+```markdown
+October 25, 2016 (11:45 UTC)
+```
+
+## Game modes
+
+Game modes must be written as follows:
+
+- `osu!standard` (unofficial, but used to prevent ambiguity)
+- `osu!taiko`
+- `osu!catch`
+- `osu!mania`
+
+*Notice: `osu!standard` is used to maintain consistency when referring to the game mode. Folder names, however, must use `osu!` even if it is referring to the game mode.*
```

Comment:
I think it's a note, not notice

Reply:
`Note` in the ASC is intended to point out some trivial information. Though, I have been seeing some signs that this does not make much sense, is sometimes confusing, and may depend on who is writing/reading it.

I guess I'm now leaning towards removing `Notice` in favor of `Note`, or change `Note` to `Trivia` and `Notice` to `Note` though I think this should be discussed with a few others first.

Reply:
to me, "Trivia" is equally as meaningless as "Notice" and could be covered by just "Note", but that's for another PR to not keep this one hanging for any longer